### DISCUSSION TITLE
why english translation is so small in the dictionary

### DISCUSSION THREAD

**Discussion by serene-ding** [Jitendex/Jitendex#101] [Q&A] [answered]
can the appearance be changed. couldnt find an option to adjust the font of the dictionary example sentences.
![image](https://github.com/user-attachments/assets/53006d35-5ce7-47a8-87b0-bd12855d3cc7)

**Comment by stephenmk** [comment, marked as answer]
Answered in [yomitan #1304](https://github.com/themoeway/yomitan/issues/1304)
